blas64/blas64.go

```diff
@@ -140,33 +140,35 @@ func Iamax(n int, x Vector) int {
 	return blas64.Idamax(n, x.Data, x.Inc)
 }
 
-// Swap exchanges the elements of two vectors.
-//  x[i], y[i] = y[i], x[i] for all i
+// Swap exchanges the elements of the two vectors:
+//  x[i], y[i] = y[i], x[i] for all i.
 func Swap(n int, x, y Vector) {
 	blas64.Dswap(n, x.Data, x.Inc, y.Data, y.Inc)
 }
 
-// Copy copies the elements of x into the elements of y.
-//  y[i] = x[i] for all i
+// Copy copies the elements of x into the elements of y:
+//  y[i] = x[i] for all i.
 func Copy(n int, x, y Vector) {
 	blas64.Dcopy(n, x.Data, x.Inc, y.Data, y.Inc)
 }
 
-// Axpy adds alpha times x to y
-//  y[i] += alpha * x[i] for all i
+// Axpy computes
+//  y = alpha * x + y,
+// where x and y are vectors, and alpha is a scalar.
 func Axpy(n int, alpha float64, x, y Vector) {
 	blas64.Daxpy(n, alpha, x.Data, x.Inc, y.Data, y.Inc)
 }
 
-// Rotg computes the plane rotation
```

Comment:
From my point of view the description of `r`, `c` and `s` was redundant because they are already fully described by the matrix/vector relation and the fact that the matrix is a rotation.